src/pages/steps/LocationStep.tsx

```diff
@@ -58,6 +68,92 @@ const API_URL = publicRuntimeConfig.apiUrl;
 
 const getGlobalValue = getValueFromTheme('global');
 
+const RecentLocations = ({ onFieldChange, ...props }) => {
+  const { t, i18n } = useTranslation();
+  const getUserQuery = useGetUserQuery() as { data: User };
+  const getOffersQuery = useGetOffersByCreatorQuery(
+    {
+      advancedQuery: '_exists_:location.id',
+      creator: getUserQuery?.data,
+      sortOptions: {
+        field: 'modified',
+        order: 'desc',
+      },
+      paginationOptions: { start: 0, limit: 20 },
+    },
+    {
+      queryArguments: {
+        workflowStatus: 'DRAFT,READY_FOR_VALIDATION,APPROVED',
+        addressCountry: '*',
+      },
+    },
+  ) as { data: { member: (Offer & { location: any })[] } };
```

Comment:
I know this is a bit hacky but it seems `UseAuthenticatedQueryResult` is currently not quite working since all the other places that use these queries have to resort to `ts-expect-error` every time :/

Reply:
Yes you're correct, this is something we can improve on when we have time.

There is already a PR in draft (has been for a while)

To improve the typing of our react-query wrappers:

https://github.com/cultuurnet/udb3-frontend/pull/393

Reply:
So for now casting or `ts-expect-error` is fine 👍🏻